pallets/automation-time/src/lib.rs

```diff
@@ -1345,10 +1357,28 @@ pub mod pallet {
 				Self::is_valid_time(*time)?;
 			}
 
+			// Execution fee
 			let fee =
 				Self::calculate_execution_fee(&action, execution_times.len().try_into().unwrap());
-			T::FeeHandler::can_pay_fee(&who, fee.clone())
-				.map_err(|_| Error::<T>::InsufficientBalance)?;
+
+			// XCMP fee
+			let xcmp_fee: u128 = match action {
+				Action::XCMP { para_id, currency_id, encoded_call_weight, .. } =>
+					T::XcmpTransactor::get_xcm_fee(
+						u32::from(para_id),
+						currency_id,
+						encoded_call_weight.clone(),
+					)
+					.unwrap()
+					.saturating_mul(execution_times.len().try_into().unwrap()),
+				_ => 0u32.into(),
+			};
+
+			T::FeeHandler::can_pay_fee(
+				&who,
+				fee.clone().saturating_add(<BalanceOf<T>>::saturated_from(xcmp_fee.clone())),
```

Comment:
what if the xcmp fee is in a different currency than the normal fee? does that have to get factored in?

Reply:
we only accept in TUR (Native) right now.  https://github.com/OAK-Foundation/OAK-blockchain/blob/master/pallets/xcmp-handler/src/lib.rs#L170-L189

Reply:
prob just toss a todo then to make sure that we don't forget if/when we add another currency!